src/app/static/src/tests/components/plots/bubble/bubblePlot.test.ts

```diff
@@ -55,59 +55,64 @@ const allAreaIds = ["MWI", "MWI_3_1", "MWI_3_2", "MWI_4_1", "MWI_4_2"];
 
 const getWrapper = (customPropsData: any = {}) => {
 
-    return shallowMount(BubblePlot, {propsData: {...propsData, ...customPropsData}, localVue});
+    return shallowMountWithTranslate(BubblePlot, store, {
+        props: {...props, ...customPropsData},
+        global: {
+            plugins: [store]
+        }
+    });
 };
 
-export const expectIndicatorSelect = (wrapper: Wrapper<Vue>, divId: string, value: string) => {
+export const expectIndicatorSelect = (wrapper: any, divId: string, value: string) => {
```

Comment:
hmmm I think I should just fix it here because I changed it in this PR! But I agree, so I think the reason was because now there is a distinction between DOMWrapper and VueWrapper and so we needed to be either more careful with types and perhaps allow either one in some cases or we could do `any`s but I am happy to tighten up the `any`s!